src/themes/ec/maps/color.scss

```diff
@@ -287,9 +287,9 @@ $color-modes: (
     '--cm-surface-low-1': map.get($color-palette, 'blue-100'),
     '--cm-surface-low-2': map.get($color-palette, 'blue-200'),
     '--cm-surface-medium': map.get($color-palette, 'blue-300'),
-    '--cm-surface': map.get($color-palette, 'blue-500'),
+    '--cm-surface': map.get($color-palette, 'blue-navy-900'),
```

Comment:
the blue color mode is different than the default one; this is not supposed to change here

Reply:
done, thanks